Lib/textwrap.py

```diff
@@ -139,6 +145,7 @@ def __init__(self,
         self.max_lines = max_lines
         self.placeholder = placeholder
 
+        self.len = cjklen if self.cjk else len
```

Comment:
I suggest to make it private and use a different name than len: self._text_width() for example.